web-app/src/services/API/internalAPI/axiosInstance.ts

```diff
@@ -18,9 +18,8 @@ const handleResponseErrors = (response: AxiosResponse) => {
     | number;
 
   // TODO: check cookie
-  const currentlyLoggedIn = !isNil(
-    'TokenManager.getAccessToken() - check cookie here'
-  );
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+  const currentlyLoggedIn = !isNil(response.headers['set-cookie']);
```

Comment:
Here check cookie in browser, not from response/request
https://github.com/cmp-cc/vue-cookies#api

Reply:
Yeah that seemed strange to me as well but you prepared this strange "schema" and I didn't really know what it was about.